### ISSUE TITLE
Between extrusion role change G-Code (Dynamic PA macro)

### ISSUE THREAD

**Issue by kubkadl** [SoftFever/OrcaSlicer#2339]
**Is your feature request related to a problem? Please describe.**
No

**Which printers will be beneficial to this feature**
E.g. Voron/Klipper based printer/etc...

**Describe the solution you'd like**
Add custom between extrusion role change g-code like it is in SS

**Additional context**
I would like to add this custom g-code to use macros for dynamic PA (https://github.com/hadesz/v2blorgr-config/wiki/Dynamic-Pressure-Advance)

**Comment by Ocraftyone** [comment]
That's a really cool idea! Seen as how SS already implemented that, it's just a matter of finding the important parts in SS and transferring them to orca. I'll take a look into doing this when I get a chance.

**Comment by kubkadl** [comment]
> That's a really cool idea! Seen as how SS already implemented that, it's just a matter of finding the important parts in SS and transferring them to orca. I'll take a look into doing this when I get a chance.

Have you watched it yet?

**Comment by Ocraftyone** [comment]
Yup. Just took a look at it and implementation was pretty simple. I am building a test build and if everything goes smooth, I'll submit a PR
